### PR TITLE
Add JavaScript code example

### PR DESCRIPTION
**Pull request by TimvdLippe**
Fixes the JS part of #6

### CODE REVIEW

**Comment by skamoen** [comment]
Could you add how to obtain (and redirect) to Mollie after the post request?

Also, thanks for removing those nasty whitespaces!

File: README.md [hunk 1]
```diff
@@ -31,23 +31,42 @@ print(response.json())
 ```
 
 #### JavaScript
```
Comment (martijnjanssen) [inline README.md:33]: Maybe add a note that fetch is not supported everywhere yet?

File: README.md [hunk 2]
```diff
@@ -31,23 +31,42 @@ print(response.json())
...
+fetch('http://localhost:9000/api/orders', {
+  method: 'POST',
+  body: {
+    name: 'Thomas Ticket',
+    email: 'thomasticket@example.com',
+    returnUrl: 'https://www.ch.tudelft.nl/payments/ordercompleted',
+    productKeys: ['e44685c5-a360-467c-960d-29843a101bb1', '879941f4-43d1-4ff2-ad66-cbdf3b141cab']
+  }
+});
+```
```
Comment (martijnjanssen) [inline README.md]: I'd add a print statement for the result of the request, this way the user is immediately able to see what the response contains.

File: README.md [hunk 3]
```diff
@@ -31,23 +31,42 @@ print(response.json())
...
+
+Obtaining the status:
```
Comment (martijnjanssen) [inline README.md]: Should we also add this to the python section?

**Review by martijnjanssen — COMMENTED** [review]
Looks good.

File: README.md [hunk 4]
```diff
@@ -31,23 +31,42 @@ print(response.json())
...
+```js
+const request = await fetch('http://localhost:9000/api/orders/status/e44685c5-a360-467c-960d-29843a101bb1');
```
Comment (jgadelange) [inline README.md]: I think this example should use the order id from the request you have done above

File: README.md (hunk 3 above)
Comment (TimvdLippe) [reply README.md]: Yes :smile:

**Comment by TimvdLippe** [comment]
The snippet was updated according to your feedback :smile:

**Comment by skamoen** [comment]
We may need to explain it in sjaars-terms, but let's leave that for a dedicated "code example" page.

**Pull request merged**